phpBB/styles/prosilver/theme/responsive.css

```diff
@@ -557,14 +556,14 @@
 	.postbody { width: 70%; }
 }
 
-@media (min-width: 850px) {
-	.postbody { width: 76%; }
-}
-
-@media (max-width: 850px) {
-	.postprofile { width: 28%; }
-
+@media (min-width: 701px) and (max-width: 850px) {
+	.postbody {
+		width: 76%;
+	}
 
+	.postprofile {
+		width: 22%;
+	}
 }
```

Comment:
this is a failed implementation. The postprofile should not be shrunk in-fact as I stated in the post on .com it should be hardcoded 200px with a 10px left margin and the postbody should be scaled down when necessary. This is why I said it would require a complete refactor for the responsiveness of these two elelements.

Reply:
it will probably require 3 or adjustments under 1100px or so

Reply:
Okay, so what is it you want me to do here?
You obviously have something specific in mind?

And isn't hardcoding the width to 200px and then setting the `.postbody` to `calc(100% - 210px)` an option?

Reply:
Yes you can do that in 3.3 or 4.0, but still have to factor out all the other stuff. But 3.2 still supports ie 10/11 and technically 3.3 still supports 11 partially which has problems with calc